ArduCopter/Copter.h

```diff
@@ -777,6 +777,8 @@ class Copter : public AP_HAL::HAL::Callbacks {
 
     // landing_gear.cpp
     void landinggear_update();
+    bool gcs_gear_sw_warned = false;
```

Comment:
No need to initialise here.  It's done magically by the program startup sequence.